app/src/main/java/com/devindi/wallpaper/model/map/TileUtils.kt

```diff
@@ -0,0 +1,50 @@
+package com.devindi.wallpaper.model.map
+
+import com.devindi.wallpaper.misc.RectD
+import org.osmdroid.tileprovider.cachemanager.CacheManager
+import timber.log.Timber
+
+class TileUtils {
+
+    fun getTilesRect(centerLat: Double, centerLon: Double, width: Int, height: Int, tileSize: Int, zoom: Int): RectD {
+        val centerTileY = latToTile(centerLat, zoom)
+        val centerTileX = lonToTile(centerLon, zoom)
+        Timber.d("Generating bounding box: center %f %f", centerLat, centerLon)
+        Timber.d("Generating bounding box: center $centerTileX $centerTileY")
+        Timber.d("Generating bounding box osm: center ${CacheManager.getMapTileFromCoordinates(centerLat, centerLon, zoom)}")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/63cb55c913b0d690492f74a58d6634a0c41df234/app/src/main/java/com/devindi/wallpaper/model/map/TileUtils.kt#L14"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>